backend/src/main/java/gov/cdc/usds/simplereport/api/accountrequest/AccountRequestController.java

```diff
@@ -164,28 +192,34 @@ public AccountResponse submitAccountRequestV2(@Valid @RequestBody AccountRequest
     }
   }
 
-  private Map<String, String> convertAccountRequestToMap(AccountRequest accountRequest)
-      throws JsonProcessingException {
-    if (LOG.isInfoEnabled()) {
-      LOG.info("Account request submitted: {}", objectMapper.writeValueAsString(accountRequest));
-    }
+  /** Organization Account request for experian id verification workflow. */
```

Comment:
Can you clarify this comment a little more, that it's for organization requests without facility?